soc/nxp/imxrt/imxrt11xx/soc.c

```diff
@@ -55,22 +55,19 @@
 #endif
 
 #ifdef CONFIG_INIT_ARM_PLL
+/*
+ * Check that the ARM PLL has a multiplier and divider set
+ */
+BUILD_ASSERT(DT_NODE_HAS_PROP(DT_NODELABEL(arm_pll), clock_mult),
```

Comment:
instead of build assert, maybe just do DT_PROP_OR to default to what was hardcoded before

Reply:
Fine with that- I was trying to drop the SOC specific code here but defaulting back to the previously set values will avoid breaking users downstream